packages/tokenbank_v1/contracts/impl/BaseWallet.sol

```diff
@@ -69,7 +77,7 @@ contract BaseWallet is Wallet, NamedAddressSet, ReentrancyGuard
         require(_modules.length > 0, "EMPTY_MODULES");
 
         owner = _owner;
-
+        bankRegistry.registerWallet(address(this));
```

Comment:
I think this will have to be moved to `WalletFactory` because `registerWallet` is `onlyFactory` (which makes sense).

Reply:
You are right! Fixed